security/fma-holocene.md

```diff
@@ -167,19 +167,37 @@ The changes to the MIPs contract were subject to an audit, the report is availab
 Below is what needs to be done before launch to reduce the chances of the above failure modes occurring, and to ensure they can be detected and recovered from:
 
 - [x] (BLOCKING): Resolve all comments on this document and incorporate them into the document itself (Assignee: document author)
-- [ ] (BLOCKING): Action tests will be added which are run on op-node and Kona https://github.com/ethereum-optimism/optimism/issues/12449
-- [ ] (BLOCKING): The changes will be deployed to a devnet with both geth and reth running
-- [ ] (BLOCKING): we will re-architect the batcher to eliminate the cases above
-- [ ] (BLOCKING): we will add functionality to the batcher to periodically detect when the safe chain should have progressed but didn't, and to recover from that
+- [x] (BLOCKING): Action tests will be added which are run on op-node and Kona 
+    * [Holocene-D: Action Tests](https://github.com/ethereum-optimism/optimism/issues/12449)
+    * [op-e2e/actions: Add Holocene FP action tests #12520](https://github.com/ethereum-optimism/optimism/pull/12520)
+    * [op-e2e/actions: Cover Holocene activation and pipeline reset #12891](https://github.com/ethereum-optimism/optimism/pull/12891)
+    * [chore(ci): Bump monorepo commit](https://github.com/anton-rs/kona/pull/805)
+
+- [x] (BLOCKING): The changes will be deployed to a devnet with both geth and reth running
+    * [Add Holocene activation timestamps for Sepoila devnets and testnets](https://github.com/ethereum-optimism/superchain-registry/pull/677)
+    * [chore: Update Holocene timestamp for Sepolia](https://github.com/paradigmxyz/reth/pull/12479)
+    * Reth does not have a chain spec built in for devnet, so a custom one will be used
+    * TODO link to op-node/op-geth/reth activation releases and k8s PR for rollout
+
+- [x] (BLOCKING): we will re-architect the batcher to eliminate the cases above
+    * [Holocene-D: Batcher Hardening](https://github.com/ethereum-optimism/optimism/issues/12120)
+    * [batcher: keep blocks, channels and frames in strict order & simplify reorg handling #12390](https://github.com/ethereum-optimism/optimism/pull/12390)
+    * [op-batcher: add channel.straddlesActivation method (for Holocene) and wire it up #12743](https://github.com/ethereum-optimism/optimism/pull/12743)
+- [x] (BLOCKING): we will add functionality to the batcher to periodically detect when the safe chain should have progressed but didn't, and to recover from that
+    * [batcher: batchSubmitter.checkExpectedProgress #12430](https://github.com/ethereum-optimism/optimism/pull/12430)
 - [ ] (BLOCKING): the script(s) or tooling for the `SystemConfig` upgrade contract will exit with an error if the activation time has not yet passed.
+    * The scripts are here https://github.com/ethereum-optimism/optimism/pull/12878 but there is no check on the activation time
```

Comment:
This is unresolved for now cc @clabby

Reply:
In the end it was decided that it is not necessary to (needlessly cautious to) implement this check, so we can remove it from the TODOs. Instead we split the superchain ops tasks into pre and post activation, and don't execute the post activation until afterwards.